lambda/create-investigation/conftest.py

```
@@ -10,6 +10,46 @@
+import pytest
+
 _this_dir = os.path.dirname(os.path.abspath(__file__))
 if _this_dir not in sys.path:
     sys.path.append(_this_dir)
+
+# Pin a closed credential environment BEFORE test_handler imports handler.
+# Do not clear keys and leave the default chain open — that can still resolve
+# ambient ~/.aws, IMDS, or container credentials. Use explicit test-only keys
+# (moto convention) and disable every ambient provider.
+os.environ['AWS_ACCESS_KEY_ID'] = 'testing'
+os.environ['AWS_SECRET_ACCESS_KEY'] = 'testing'
```

Ruff (0.16.2)

[error] Possible hardcoded password assigned to: "AWS_SECRET_ACCESS_KEY"

(S105)